solutions/usaco-639.mdx

```diff
@@ -50,4 +50,32 @@ int main(){
 
 </CPPSection>
 
+<PySection>
+
+```py
+fin = open('diamond.in', 'r')
+
+N, K = map(int, fin.readline().split())
+
+diamonds = []
+
+for _ in range(N):
+    diamonds.append(int(fin.readline()))
+
+tracker = []
+largest, curlargest = 0
```

Comment:
this should error?

Reply:
i just realized that earlier, it's fixed now.